input/sources/tev-000155.yaml

```diff
@@ -1,11 +1,25 @@
 source_id: 155
 
+common_name: H 2356-309 # "canonical name" row on tevcat website. Also listed in tevcat.ecsv, "source_name" column
+gamma_names: [] # ?
+other_names: [1H 2354-315, HESS J2359-306] # "Other names" row on tevcat's website. Also listed in "other_names" column in tgevcat.ecsv
```

Comment:
Put the HESS name in `gamma_names` instead.
`gamma_names` should be for sources detected by gamma-ray instruments, i.e. with prefix HESS, VER (VERITAS) or MAGIC or HAWC.
It's OK to leave this empty for sources where you don't have such a name.